spec/collectors/statsd.rb

```diff
@@ -73,6 +73,11 @@ module Collectors
           str = collector.build_stamp('There be dragons', 101)
           expect(str).to eq '101_0de17114_There_be_dragons'
         end
+
+        it 'Prefixes git commit hash with commit date' do
+          result = collector.results.first.description.match(/\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}/)
```

Comment:
Because of lack of using Match object you may simply use `=~` like:
```ruby
result = collector.results.first.description =~ /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}/
```

Reply:
The regex should also have `$` and `^` symbols to avoid matching to any string with formatted timestamp inside

Reply:
That is a good idea to tighten up this assertion.  Unfortunately the description is the entire stat path so the date string is in the middle.  Example: "spec.ldxe.app.rspec_profiling.datestring_commithash.spec.test.shared.lib.controller.rb"